### PR TITLE
Fix for CF cli not displaying route service url on "cf service"

### PR DESCRIPTION
**Pull request by jeffleecoffee**
## Does this PR modify CLI v6 or v7?
CLI v6

## What Need Does It Address?
This can be used to debug issues or view the service route URL of a user created service.
Example: "cf cups test_service -r https://<url>". This URL should be shown for that service's detail "cf service test_service"

## Who Is The Functionality For?
This will be to address the bug issued here: https://github.com/cloudfoundry/cli/issues/1506

## How Often Will This Functionality Be Used?
This will be used specifically for "cf service" on user provided services. If routes are not present then the display will show the field, but no results (similar to tags).

## Possible Drawbacks
No potential drawbacks, other than extra clutter to display the URL.

## Why Should This Be In Core?
This should a basic functionality of the service information summary. Otherwise, the end user would be using "cf service <name> -v" to find an essential component of service use.

## Description of the Change
As indicated in the initial Github issue, the Route service URL property was being returned from the cloud controller to the CLI. This commit just exposes the detail for user provided service summaries.

## Alternate Designs
For this addition to the view, due to simplicity, no alternatives were considered.

## Applicable Issues
None.

## How Urgent Is The Change?
Not urgent, as there is a work around to use "cf service <name> -v"

## Other Relevant Parties
None.

### CODE REVIEW

**Comment by AartiKriplani** [comment]
@jeffleecoffee thanks for the PR, changes look ok but would be good to have some tests. At the least on unit level `service_instance_test.go` to make sure that we're serializing the new field.

**Comment by jeffleecoffee** [comment]
@AartiKriplani thank you for the recommendation and where they can be added. I've added some unit test checks to watch the field.

**Comment by AartiKriplani** [comment]
@jenspinney @tjvman : LGTM, could you please review this PR and merge if it's fine ? Thanks!

File: command/v6/service_command.go [hunk 1]
```diff
@@ -183,6 +183,7 @@ func (cmd ServiceCommand) displayUserProvidedServiceInstanceSummary(serviceInsta
 	table := [][]string{
 		{cmd.UI.TranslateText("name:"), serviceInstanceSummary.Name},
 		{cmd.UI.TranslateText("service:"), cmd.UI.TranslateText("user-provided")},
+		{cmd.UI.TranslateText("route service url:"), serviceInstanceSummary.RouteServiceURL},
```
Comment (AartiKriplani) [inline command/v6/service_command.go]: @jeffleecoffee : Would be nice to have a unit test for this in `service_command_test.go` too. Thanks

Comment (jeffleecoffee) [reply command/v6/service_command.go]: Good idea, I've made some changes to only print routes when the service has it and also added an extra unit test for service_command_test.

**Comment by abbyachau** [comment]
Hey @AartiKriplani let us know if we are able to merge this pull request - we'll wait on your ok (it looks ok to us) before we do so. Thanks!

**Comment by AartiKriplani** [comment]
LGTM, please go ahead and merge @abbyachau , thanks

**Comment by jeffleecoffee** [comment]
Thanks both @abbyachau and @AartiKriplani

**Pull request merged**